src/status_im2/contexts/shell/jump_to/components/switcher_cards/view.cljs

```diff
@@ -13,10 +13,25 @@
             [status-im2.contexts.shell.jump-to.components.switcher-cards.style :as style]
             [status-im2.contexts.chat.messages.resolver.message-resolver :as resolver]))
 
+(defn- channel-card
+  [{:keys [emoji channel-name customization-color] :as _community-channel}]
+  [rn/view style/channel-card-container
+   [quo/channel-avatar
+    {:emoji               emoji
+     :customization-color customization-color}]
+   [rn/view style/channel-card-text-container
+    [quo/text
+     {:size            :paragraph-2
+      :weight          :medium
+      :number-of-lines 1
+      :ellipsize-mode  :tail
+      :style           style/community-channel}
+     channel-name]]])
```

Comment:
I extracted the channel card and fixed it, previously was defined in the function's `case` below.